resources/views/emails/quote.blade.php

```diff
@@ -538,14 +538,26 @@
                                             </p>
 
                                             @if(!empty($quote->loading_dock_instructions))
-                                            <p>
-                                              <b>Loading Dock Instructions</b>: {{$quote->loading_dock_instructions}}
-                                            </p>
-                                          @endif
+                                              <p>
+                                                <b>Loading Dock Instructions</b>: {{$quote->loading_dock_instructions}}
+                                              </p>
+                                            @endif
 
                                             <p>
                                               <b>Message</b>: {{$quote->message}}
                                             </p>
+                                            <p>
+                                              @php
+                                                $productUrl = array_get($quote->seo_metrics, 'product_url');
```

Comment:
Pass the `productUrl` from the controller and remove it from here

Reply:
As discussed, change not required.